projects/resettlement-passport-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/CaseNoteService.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.api.model.CreateContact
+import uk.gov.justice.digital.hmpps.audit.service.AuditableService
+import uk.gov.justice.digital.hmpps.audit.service.AuditedInteractionService
+import uk.gov.justice.digital.hmpps.entity.*
+
+@Service
+class CaseNoteService(
+    auditedInteractionService: AuditedInteractionService,
+    private val caseNoteRepository: CaseNoteRepository,
+    private val personRepository: PersonRepository,
+    private val personManagerRepository: PersonManagerRepository,
+    private val caseNoteTypeRepository: CaseNoteTypeRepository
+) : AuditableService(auditedInteractionService) {
+
+    fun createContact(crn: String, createContact: CreateContact) {
```

Comment:
This is extending the AuditableService, but not doing any auditing.  You just need to wrap your method call with a call to `audit` e.g.
```suggestion
) : AuditableService(auditedInteractionService) {

    fun createContact(crn: String, createContact: CreateContact) = audit(BusinessInteractionCode.ADD_CONTACT) {
```

Reply:
done!